lib/helperFunctions.tsx

```diff
@@ -5,42 +5,99 @@ import { CreatorContext } from "src/context"
 import { constants } from "./constants"
 import Link from "next/link"
 import { Card } from "src/components"
+const data: Data = require("data/data-videos.json")
+
+
+interface Data {
+  [key: string]: {
+    active: any
+    creatorOne: {
+      active: boolean
+      items: {
+        kind: string
+        etag: string
+        id: {
+          kind: string
+          videoId: string
+        }
+        snippet: {
+          imgCreator:string
+          publishedAt: string
+          channelId: string
+          title: string
+          description: string
+          thumbnails: any
+          channelTitle: string
+          liveBroadcastContent: string
+          publishTime: string
+        }
+      }[]
+    }
+  }
+}
+
+export const SubscribedSectionLine = () => {
+  const context = useContext(CreatorContext)
 
-export const SubscribedSectionLine: any = () => {
 
 
-  
-  let subscribedVideos: VideoType[] = getVideos("")
+  const subscribedVideos = (): VideoType[] => {
+    let activeVideos: VideoType[] = []
 
-  const context = useContext(CreatorContext)
-  function subscribedContent() {
-    context.setState(constants.creatorOne.state)
-    context.setCreator(constants.creatorOne.name)
-    context.setPayment(constants.creatorOne.paymentLink)
-    context.setImage(constants.creatorOne.image)
-    context.setSubscription(constants.creatorOne.subscription)
+    for (const creator in data) {
+      const creatorData = data[creator]
+      if (creatorData.active) {
+        const videos = getVideos(creator)
+        for (const video of videos) {
+          activeVideos.push(video)
+        }
+      }
+    }
+
+    for (let i = activeVideos.length - 1; i > 0; i--) {
```

Comment:
I am not sure this will be sustainable on the long run

Reply:
It would be too slow?